packages/file_selector/file_selector/example/lib/readme_standalone_excerpts.dart

```diff
@@ -0,0 +1,57 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This file exists solely to host compiled excerpts for README.md, and is not
+// intended for use as an actual example application.
+
+// ignore_for_file: public_member_api_docs
+
+// #docregion Save
+import 'dart:typed_data';
```

Comment:
There's an incoming PR ignoring `dart:typed_data` as an unneeded import. Do we need that ignore here too?

Reply:
Good call. I think I'll just leave this out of the README then to avoid issues where it becomes stale, or where the readme has an // ignore.